geonames/track.json

```diff
@@ -297,6 +312,27 @@
           "warmup-iterations": 500,
           "iterations": 1000,
           "target-throughput": 20
+        },
+        {
+          "operation": "large_terms",
+          "clients": 1,
+          "warmup-iterations": 500,
+          "iterations": 1000,
+          "target-throughput": 50
```

Comment:
Based on the relation of latency and service_time I've seen in my tests I'd suggest to lower that setting to 3 ops/s in order to not over-saturate the system on our current hardware.